da-3-hedonensistractatus-projectdata.xml

```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<listofFileNames>
+  <header>
+    <authorName>Guillelmus Hedonensis</authorName>
+    <commentaryName>Tractatus de anima</commentaryName>
+    <commentaryid></commentaryid>
+    <commentaryslug>3</commentaryslug>
+    <authorUri></authorUri>
+  	<parentUri></parentUri>
+  	<textfilesdir></textfilesdir>
+    <webbase>null</webbase>
+    <questionListSource></questionListSource>
+    <questionListOriginalEditor>Michael Stenskjær Christensen</questionListOriginalEditor>
+    <questionListEncoder>Michael Stenskjær Christensen</questionListEncoder>
+    <hasWitnesses>
+      <witness id="C">
+        <slug>cam342</slug>
+        <title>Cambridge, Gonville & Caius College, 342/538</title>
+        <initial>C</initial>
+      </witness>
+      <witness id="O">
+        <slug>oxf107</slug>
```

Comment:
we could stick with oxf107, but it might be better to do colleges, since each college will have a 107. I would suggest corpchri107. But again it doesn't matter. Display results will normally get sorted and display by codex title rather than id, so all the Oxford mss will get grouped together no matter what the id is. So really, you can just leave this.